### DISCUSSION TITLE
synthesis problem

### DISCUSSION THREAD

**Discussion by jimmy20000314** [bol-edu/HLS-SOC-Discussions#141] [Q&A] [answered]
想請問一下synthesis後沒有讀到clk(有設constraint了)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145492034/3fadf1bd-6b6b-4c22-8317-df90016d8263)

warning:
`CRITICAL WARNING: [Common 17-165] Too many positional options when parsing 'axis_clk', please type 'create_clock -help' for usage info. [D:/SOC/course-lab_3-2022.1/fir/constrc_1.xdc:1]`

研究了一 下發現constraint的clk沒有成功get到
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145492034/3f7923c2-c86f-46ca-a102-f7cf63477097)
這個是甚麼原因呢?

**Comment by edwardchang9** [comment, marked as answer]
他說你的create_clock指令的argument太多，或許是你的- periods、- name、- waveform的-跟後面多加了空格?應為-period、-name、-waveform

**Reply by jimmy20000314** [reply]
喔原來是格式的問題，我發現除了period多個s外，-的格式錯了(半形全形)，感謝你!!成功跑出時間了!!!
